tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -68,7 +69,13 @@ function createUser (userId, password, displayName = false, email = false) {
 function deleteUser (userId) {
   const headers = httpHelper.createAuthHeader(client.globals.backend_admin_username)
   userSettings.deleteUserFromCreatedUsersList(userId)
-  return fetch(backendHelper.getCurrentBackendUrl() + '/ocs/v2.php/cloud/users/' + userId, { method: 'DELETE', headers: headers })
+  return fetch(
+    join(
+      backendHelper.getCurrentBackendUrl() +
+      '/ocs/v2.php/cloud/users/' +
+      userId
+    ),
```

Comment:
Why are you using join, and concatenating inside it? Then, what's the use of `join` here?

Reply:
didn't realized ok fixing.